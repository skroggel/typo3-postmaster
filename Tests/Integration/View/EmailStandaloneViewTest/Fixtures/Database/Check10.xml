<?xml version="1.0" encoding="utf-8"?>
<dataset>
    <pages>
        <uid>10</uid>
        <pid>0</pid>
        <title>Rootpage</title>
        <doktype>1</doktype>
        <perms_everybody>15</perms_everybody>
    </pages>
    <pages>
        <uid>11</uid>
        <pid>10</pid>
        <title>Test</title>
        <doktype>1</doktype>
        <perms_everybody>15</perms_everybody>
    </pages>
</dataset>