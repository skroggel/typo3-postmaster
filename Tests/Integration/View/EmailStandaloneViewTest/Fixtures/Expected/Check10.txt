

<div id="8effee409c625e1a2d8f5033631840e6ce1dcb64-99" class="search-result-section" data-ajaxapi-id="99" data-ajaxapi-action="replace">

        <div class="boxes-grid" id="grid-section">

            <div class="box-wrapper">
                <a class="box topic-3 event big" title="Roadshow Tournee.Digital in Baden-Württemberg an der österreichischen Weinstraße" href="veranstaltungen/details/roadshow-tourneedigital-in-sachsen-anhalt-552/">
                    <header>
                        <p>

                            Roadshow Tournee.Digital in Baden-Württemberg an der österreichischen Weinstraße
                        </p>
                    </header>
                    <div class="box-content">
                        <p class="event-city">Haldensleben</p>
                        <p class="event-date">
                            <strong>
                                01.09.2020
                            </strong>
                        </p>
                    </div>
                    <footer>
                        <p class="icon-calendar">Veranstaltung</p>
                    </footer>
                </a>
            </div>
        </div>

    </div>